components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-overview.tsx

```diff
@@ -194,24 +222,62 @@ export const PortfolioOverview = () => {
     const grandTotal = visibleAssetFiatBalances.reduce(function (a, b) {
       return a.plus(b)
     })
-    return grandTotal.formatAsFiat(defaultCurrencies.fiat)
+    return grandTotal.format()
   },
     [
       visibleAssetOptions,
-      defaultCurrencies.fiat,
       transactionSpotPrices
     ])
 
+  const formattedFullPortfolioFiatBalance = React.useMemo(() => {
+    return fullPortfolioFiatBalance !== ''
+      ? new Amount(fullPortfolioFiatBalance)
+        .formatAsFiat(defaultCurrencies.fiat)
```

Comment:
See my previous comment - you should be able to do `!fullPortfolioFiatBalance.isUndefined()` and also avoid using the `Amount` constructor again.

Reply:
Updated!